src/views/LandingPage/LandingPage.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+
+import { Button } from "shards-react";
+import TitleLogo from "../../components/TitleLogo";
+import RegisterForm from "../../components/RegisterForm";
+import LoginForm from "../../components/LoginForm";
+import "./styles.scss";
+
+/**
+ * @description container component that should be in the middle of the login page
+ */
+
+export default props => {
+  const [formType, setFormType] = useState("register");
+
+  return (
+    <div className="page__register">
+      <div className="title-logo">
+        <TitleLogo />
+      </div>
+      <div className="register__container">
+        {formType === "register" ? <RegisterForm /> : <LoginForm />}
+        <div>
+          <h6>
+            {formType === "login" ? "New to Billsnap? " : "Have an account? "}
```

Comment:
Why dont you add these texts in the `LoginForm` and `RegisterForm` components respectively?